[DOC-450][ybm] CLI example for modifying cluster scale (#26606)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md

my-multi-region   Dedicated   2.14.7.0-b51   ACTIVE    💚        us-central1,+2   3         6 / 24GB / 600GB
```

### Scale your cluster

> Use the `ybm cluster update` command with `--region-info` flag to scale your cluster by adding or removing nodes, cores, and disk size. You can also change the region name. For example, to change the number of nodes in the `ap-northeast-1` region from 1 to 3, and the number of cores from 2 to 4, enter the following command:

Reviewer: ```suggestion
Use the `ybm cluster update` command with `--region-info` flag to scale your cluster by adding or removing nodes, cores, and disk size. You can also change the region name. For example, to change the number of nodes in the `us-east1` region from 1 to 3, and the number of cores from 2 to 4, enter the following command:
```

```sh
ybm cluster update \
  --cluster-name my-multi-region \

>   --region-info region=us-east1,num-nodes=3,num-cores=4,disk-size-gb=250 \

Reviewer 2: ```suggestion
  --region-info region=ap-northeast-1,num-nodes=3,num-cores=4,disk-size-gb=250 \
```

  --wait
```

> For more information, see [Scale and configure clusters](../../../../cloud-clusters/configure-clusters/)

Reviewer 2: ```suggestion
For more information, see [Scale and configure clusters](../../../../cloud-clusters/configure-clusters/).
```

Successfully ENABLED encryption at rest for cluster my-sandbox
```

> ## Scale your cluster

Reviewer: Slot this in after one of the previously created clusters - the multi region one I suppose - and actually modify the previously created cluster (ie my-multi-region). This should be an example you can actually run, but frank-wren is never created
